src/Chapter18/Listing18.26.DynamicProgrammingUsingReflection.cs

```diff
@@ -1,18 +1,23 @@
 namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter18.Listing18_26
 {
+    #region INCLUDE
     using System;
 
+    #region EXCLUDE
     public class Program
     {
         public static void Main()
         {
-            // ...
+            #endregion EXCLUDE
             dynamic data =
               "Hello!  My name is Inigo Montoya";
             Console.WriteLine(data);
             data = (double)data.Length;
             data = data * 3.5 + 28.6;
+            //
```

Comment:
```suggestion

```
Not sure what these // are for? Can it be removed or should it be made into // ...